rootfs/scheduler/__init__.py

```diff
@@ -259,13 +261,14 @@ def _get_deploy_batches(self, steps, desired):
 
         return batches
 
-    def _update_application_service(self, namespace, name, app_type, port, routable=False):
+    def _update_application_service(self, namespace, name, app_type, port, routable=False, maintenance=False):  # noqa
         """Update application service with all the various required information"""
         service = self.get_service(namespace, namespace).json()
         old_service = service.copy()  # in case anything fails for rollback
 
         try:
             # Update service information
+            service['metadata']['annotations']['router.deis.io/maintenance'] = str(maintenance)
```

Comment:
Currently the `Service` is always created when the application is, so the resource is always around, that should mean the `Service` is ready to go before a deploy. That's how `Domains` work, so I'm just making sure things are all in sync here.